app/javascript/react/ansibleCatalog/catalogActions.js

```diff
@@ -0,0 +1,84 @@
+import * as actionTypes from './actionTypes';
+import { API } from '../../http_api';
+
+export const commonParams = '&expand=resources&sort_by=name&sort_order=ascending';
+export const credentialsRequest = type => `/api/authentications?collection_class=ManageIQ::Providers::EmbeddedAnsible::AutomationManager::${type}&attributes=id,name${commonParams}`;
+const catchError = (action) => {
+  window.add_flash(__('Request failed: ') + action, 'error');
+};
+export const loadCatalogs = catalogs => ({
+  type: actionTypes.LOAD_CATALOGS,
+  payload: catalogs,
+});
+export const loadDialogs = () => dispatch => API.get('/api/service_dialogs/?expand=resources&attributes=id,label&sort_by=label&sort_order=ascending').then((data) => {
```

Comment:
You don't need the extra curly bracket around the dispatch. It is a good practice to return dispatched actions (if you have some async chaining). If you remove the brackets it will be returned automatically.

I've seen this with almost every action.